lib/core/form/services/widget-visibility.service.ts

```diff
@@ -112,7 +112,7 @@ export class WidgetVisibilityService {
         return value;
     }
 
-    getFieldValue(valueList: any, fieldId: string) {
+    getFieldValue(valueList: any, fieldId: string): any {
```

Comment:
why any ?

Reply:
Just added missing return type according to the current code